Makefile

```diff
@@ -17,10 +17,12 @@ SOURCE_GIT_TAG :=$(shell git describe --tags --abbrev=7 --match '$(RELEASE_PRE)*
 
 SRC_ROOT :=$(shell pwd)
 
-BUILD_CFG :=./images/build/Dockerfile
+BUILD_CFG :=./packaging/images/build/Dockerfile
 IMAGE_REPO :=quay.io/microshift/microshift
 OUTPUT_DIR :=_output
 CROSS_BUILD_BINDIR :=$(OUTPUT_DIR)/bin
+AIO_TAG :=$(shell git describe --tags --abbrev=0 --match '$(RELEASE_PRE)*' || echo '4.7.0-0.microshift-unknown')-aio
```

Comment:
Instead of the *-aio tag, let's look at pushing aio images to a separate repo. e.g. `quay.io/microshift/microshift-aoi:$SOURCE_GIT_TAG`.  The internal differences of the aio image vs the standard releases is enough I think to justify not pushing them to the same repo.